maintainers/maintainer-list.nix

```diff
@@ -1512,6 +1512,12 @@
     githubId = 1111035;
     name = "Break Yang";
   };
+  brecht = {
+    email = "brecht.savelkoul@alumni.lse.ac.uk";
+    github = "brechtcs";
+    githubId = 6107054;
+    name = "Brecht Savelkoul";
+  };
```

Comment:
Separate commit: `maintainers: add brecht`

Reply:
Rebased and separated the changes into two commits. It's fine to keep both of them together in one PR though, right?

Reply:
That's not an issue.